web/pandas/about/team.md

```diff
@@ -36,10 +36,19 @@ If you want to support pandas development, you can find information in the [dona
     {% endfor %}
 </div>
 
+
+## Diversity and Inclusion
+    
+> _pandas_ expressly welcomes and encourages anyone who faces under-representation,discrimination in the technology industry 
+> or anyone willing to increase the diversity of our team,we encourage you to contribute.
+> We have identified the visible diversity gap and obstacles in the open source community and we are strongly against it.
+> If you identify as any of the groups mentioned above do not hesitate to make your contribution,
+> if you see a violation of our [code of conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md) 
```

Comment:
We have our code of conduct n the web now, you can use the link `[code of conduct]({base_url}/community/coc.html)` instead.

I'd probably add a bit more context about the code of conduct. Something like `We have a [code of conduct](link) to ensure a friendly and welcoming environment. Please send us a message to ... if you don't feel this way.`

Reply:
Hello, thanks for the suggestions, 
I failed to get a link to the code of conduct used what you suggested